### ISSUE TITLE
Dog is no longer the breed it was before when revived

### ISSUE THREAD

**Issue by Clyde6790pGIT** [HeroGamers/MyDog#117]
### Type of bug

Other unexpected behaviour

### Bug description

If a dog dies, and a player attempts to revive their dog, it is now only the default breed.

### Steps to reproduce

1. Kill your dog (sad)
2. Revive it

### Expected behaviour

The dog should be the breed it was before (Woody, Snowy, Etc.)

### Actual behaviour

The doggie is just the default breed now.

**Comment by HeroGamers** [comment]
Yeah, the plugin isn't updated for the newest version yet, that should probably be fixed, otherwise the plugin kinda sucks on the new version lol

**Comment by VegasBoz** [comment]
This plugin is one of the favorites of the crew I play MC with.
